src/Core/Components/Label/FluentInputLabel.razor.css

```diff
@@ -0,0 +1,11 @@
+﻿.fluent-input-label {
+    display: block;
+    color: var(--neutral-foreground-rest);
+    cursor: pointer;
+    font-family: var(--body-font);
+    font-size: var(--type-ramp-base-font-size);
+    line-height: var(--type-ramp-base-line-height);
+    font-weight: initial;
+    font-variation-settings: var(--type-ramp-base-font-variations);
+    margin-bottom: 4px;
```

Comment:
I would use `calc(var(--design-unit) * 1px)` so it scales along if someone changes this variable

Reply:
Done